config/config_test.go

```diff
@@ -84,6 +84,7 @@ func TestGet(t *testing.T) {
 		actual, configFilepath, err := get([]string{
 			"--id=agent-id",
 			"--server-address=127.0.0.1",
+			"--paths-pt-summary=/usr/local/percona/pmm2/pt-summary",
```

Comment:
let's remove it and check default value. To learn where it placed ask @EvgeniyPatlan